jwst/tests_nightly/general/miri/test_miri_steps_single.py

```diff
@@ -284,9 +284,7 @@ def test_miri_setpointing(self):
 
         # Copy original version of file to test file, which will get overwritten by test
         input_file = self.get_data(self.test_dir,
-                                    'jw80600010001_02101_00001_mirimage_uncal_orig.fits',
-                                    docopy=True  # always produce local copy
```

Comment:
Any tests of ``set_telescope_pointing`` may need to explicitly make a copy, because they modify the input in place. Or is this OK, because we always start with a fresh copy of the original input file for each test run?

Reply:
`docopy` defaults to `True`, so it is OK.  And we always start with a fresh copy.